linux/evolution.md

```diff
@@ -1 +1,49 @@
-# Linux - Evolution
\ No newline at end of file
+# Linux - Evolution
+
+
+## History and Evolution
+
+Linux Kernel main differences among the first commit and the latest one, is that the first it's main purpose is
+for testing and no other "commercial" use, unlike the current version, which is available and stable. 29 Years after the first release, other changes are, for the 0.11 version the system must have at least 4mb of RAM, a 32 bit 
+computer. On the Software side, the newer version can manage multiple filesystem types, like ntfs, ext, btrfs, as opposed to the first one, which only supported ext filesystem. This one also supported only 3 keyboard. In summary, 
+the original spirit of the kernel remains the same, and it has been boosted with new technology, compatibility, and numerous new functions, among other things.
+
+## Evolution Chart
+
+![alt text](assets/Files-vs-Version.png "Image Example")
+
+## Architecture Decision Records
+
+1. **Title:** ADR 1: Greybus Support for version 4.9
+    1. **Context:** Greybus is intended for mobile modularity leaded by Project Ara. Google cancelled the project. 
```

Comment:
What is Greybus and Project Ara? Provide a better explanation to understand the context